torch/_inductor/codegen/cpp.py

```diff
@@ -2881,6 +2881,7 @@ def __init__(self, args, num_threads, tiling_factor, tiling_idx=-1):
             torch.int8,
             torch.int32,
             torch.int64,
+            torch.bool,
```

Comment:
with this change, we can use one list `supported_dtypes` for all such checks since the lists are all the same now. Can we define one list and used by all the checks in the code?

Reply:
Thanks for the comment. Merge into one list of `supported_dtypes`.